app/views/layouts/application.haml

```diff
@@ -66,7 +64,9 @@
       report_analytics_event.apply(null, #{@analytics_event});
   - if @gtm_datalayer
     :javascript
-      gtm_datalayer_push(#{@gtm_datalayer.to_json});
+      if(window.ST != null && typeof window.ST.gtmPush === 'function') {
```

Comment:
wouldn't `(window.ST && typeof window.ST.gtmPush === 'function')` do the same? applies also to the previous case